packages/block-library/src/post-time-to-read/index.php

```diff
@@ -21,27 +21,26 @@ function render_block_core_post_time_to_read( $attributes, $content, $block ) {
 
 	$content = get_the_content();
 
-	/*
-	 * Reading rates (words per minute) - based on averages from
-	 * https://irisreading.com/average-reading-speed-in-various-languages/
-	 * (Characters/minute used for Chinese rather than words).
-	 */
-	$average_reading_rate = 189;
-	$min_reading_rate     = 138;
-	$max_reading_rate     = 228;
+	$average_reading_rate = $attributes['averageReadingSpeed'] ?? 189;
```

Comment:
```suggestion
	/*
	 * Reading rates (words per minute) - based on averages from
	 * https://irisreading.com/average-reading-speed-in-various-languages/
	 * (Characters/minute used for Chinese rather than words).
	 */
	$average_reading_rate = $attributes['averageReadingSpeed'] ?? 189;
```